pkg/spec/cluster.go

```diff
@@ -163,6 +166,14 @@ func (cs *ClusterStatus) SetPhaseFailed() {
 	cs.Phase = ClusterPhaseFailed
 }
 
+func (cs *ClusterStatus) PausedControl() {
+	cs.ControlPaused = true
+}
+
+func (cs *ClusterStatus) Control() {
```

Comment:
if you look at the code, we actually update the status in every round. we do not really care about the previous state of `paused`. Resume assumes the previous state is paused. So it is not suitable.